Gemfile.lock

```diff
@@ -1605,9 +1595,15 @@ GEM
       net-http-persistent
     polyamorous (1.3.3)
       activerecord (>= 3.0)
-    prawn-format (0.2.1)
-      prawn-core
-    prawn-layout (0.2.1)
+    prawn (2.4.0)
+      pdf-core (~> 0.9.0)
+      ttfunk (~> 1.7)
+    prawn-markup (0.3.3)
+      nokogiri
+      prawn
+      prawn-table
+    prawn-table (0.2.2)
```

Comment:
4 years without commits, doesn't [prawn-markup](https://github.com/puzzle/prawn-markup) handle tables well enough?

Just asking, probably you already checked.

Reply:
Note to myself - prawn-markup depends on prawn-table. Also prawn-table had seen some care this year.